bundles/net.tourbook/src/net/tourbook/tour/TourInfoUI.java

```diff
@@ -1743,15 +1743,15 @@ private void updateUI() {
          _lblBreakTime.setText(breakPeriod.toString(UI.DEFAULT_DURATION_FORMATTER_SHORT));
       }
 
-      int windSpeed = _tourData.getWeather_Wind_Speed();
-      windSpeed = (int) (windSpeed / UI.UNIT_VALUE_DISTANCE);
-
-      _lblWindSpeed.setText(Integer.toString(windSpeed));
-      _lblWindSpeedUnit.setText(
-            String.format(
-                  Messages.Tour_Tooltip_Format_WindSpeedUnit,
-                  UI.UNIT_LABEL_SPEED,
-                  IWeather.windSpeedTextShort[getWindSpeedTextIndex(windSpeed)]));
+      final int windSpeed = (int) (_tourData.getWeather_Wind_Speed() / UI.UNIT_VALUE_DISTANCE);
+      if (windSpeed > 0) {
+         _lblWindSpeed.setText(Integer.toString(windSpeed));
+         _lblWindSpeedUnit.setText(
+               String.format(
+                     Messages.Tour_Tooltip_Format_WindSpeedUnit,
+                     UI.UNIT_LABEL_SPEED,
+                     IWeather.windSpeedTextShort[getWindSpeedTextIndex(windSpeed)]));
+      }
```

Comment:
It is not displayed if the wind direction is not selected
Because if the wind direction is selected to any values other than "empty", then realistically, the wind speed should not be 0
But I can add this in the `if` test

![image](https://user-images.githubusercontent.com/25235665/220688064-24fc2e87-f0d4-4a1f-9084-90a25b5b01d9.png)

Reply:
Fixed

Reply:
It looks now better

![image](https://user-images.githubusercontent.com/1283445/220834540-d9ec0c80-7ca0-41dd-bb7c-58591185e1fa.png)

Reply:
> It looks now better
> 
> ![image](https://user-images.githubusercontent.com/1283445/220834540-d9ec0c80-7ca0-41dd-bb7c-58591185e1fa.png)

Good !

Though, looking at the tooltip with all the weather information, it seems "ugly" to me that the weather info is not clear and "straight" like the info above. But I am not sure of the best solution ... thoughts ?

![image](https://user-images.githubusercontent.com/25235665/221038280-2d349d77-15ac-4cf4-8506-9a8069fcbef6.png)

Reply:
I remember that the value formatting of this part is not simple. I cannot remember the details why it is now formatted this way but as you can see in the code below just for the temperature value, it is not just a oneliner

![image](https://user-images.githubusercontent.com/1283445/221106671-1e78ca22-6369-4ede-96dd-c6fc605ec80f.png)

![image](https://user-images.githubusercontent.com/1283445/221106787-521b283f-0ced-490a-860c-4188fda823d9.png)